### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by kerlos-bahaa**
* We did fixes as mentioned in the previous Pull request 
Link: https://github.com/kerlos-bahaa/Hello-Microverse-project/pull/1
-------------------------------------------------------------------------------------------------------------------------
1- Add an HTML file with Hello Microverse! as a header

2- Add a CSS file with a color class for the header

3- Add .gitignore with node_modules and test.md included

4- Add the readme file to the project

5- Add a linter for HTML and CSS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,262 @@
...
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[your_project__name]** is a...
```
Comment (roshan-bajgain) [inline README.md]: - [ ] As requested by previous reviewer please name the project and add 1,2 line description on it for professional README.

File: README.md [hunk 2]
```diff
@@ -1 +1,262 @@
...
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
```
Comment (roshan-bajgain) [inline README.md]: - [x] Please remove other and use HTML and CSS as this project is made from HTML and CSS.

File: README.md [hunk 3]
```diff
@@ -1 +1,262 @@
...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
```
Comment (roshan-bajgain) [inline README.md]: - [ ] I would suggest you please add key feature you have on the project.

File: README.md [hunk 4]
```diff
@@ -1 +1,262 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
```
Comment (roshan-bajgain) [inline README.md]: - [ ] You can write a command on how the user can clone your project and open it in the browser. Like `git clone your url` and open through `live-server`

File: README.md [hunk 5]
```diff
@@ -1 +1,262 @@
...
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Comment (roshan-bajgain) [inline README.md]: - [ ] In this author section please use your details of Github, Twitter and LinkedIn so any user can visit your profile.

File: README.md [hunk 6]
```diff
@@ -1 +1,262 @@
...
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (roshan-bajgain) [inline README.md]: - [ ] This license is broken please fix it. There is a link on how to create an MIT license. [link](https://choosealicense.com/licenses/mit/). [Link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @kerlos-bahaa, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
- [x] You can name your pull request title `Hello Microverse`. Please do so in place of feature branch.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @kerlos-bahaa , 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**